packages/frontend-2/components/settings/workspaces/Security.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <section>
+    <div class="md:max-w-xl md:mx-auto pb-6 md:pb-0">
+      <SettingsSectionHeader title="Security" text="Be secure 😤" />
+      <div>
+        <SettingsSectionHeader title="Workspace Domains" subheading />
+        <div
+          class="flex flex-col-reverse md:justify-between md:flex-row md:gap-x-4 mt-4"
+        >
+          <FormButton @click="showAddDialog = true">Add Domain</FormButton>
+        </div>
+        <LayoutTable
+          class="mt-2 md:mt-4"
+          :columns="[
+            { id: 'domain', header: 'Domain', classes: 'col-span-3' },
+            { id: 'delete', header: 'Delete', classes: 'col-span-2' }
+          ]"
+          :items="domains"
+        >
+          <template #domain="{ item }">
+            <span class="text-body-xs text-foreground">
+              {{ `@${item.domain}` }}
+            </span>
+          </template>
+          <template #delete="{ item }">
+            <FormButton color="danger" @click="() => openRemoveDialog(item)">
+              Delete
+            </FormButton>
+          </template>
+        </LayoutTable>
+      </div>
+      <div>
+        <SettingsSectionHeader title="Domain Features" subheading class="mt-8" />
```

Comment:
Maybe add a bit of padding between these, and a `<hr />` to make it a bit more aligned with the other settings pages